fdbclient/ManagementAPI.actor.cpp

```diff
@@ -3192,6 +3199,11 @@ ACTOR Future<Void> clearBulkLoadJob(Database cx, UID jobId) {
 					    .detail("ExistJob", existJob.toString());
 					throw bulkload_task_outdated();
 				}
+				if (existJob.getPhase() == BulkLoadJobPhase::Error) {
+					TraceEvent(SevWarnAlways, "DDBulkLoadClearMetadataStopSinceJobHasErrorTask")
+					    .detail("Task", existJob.toString());
+					throw bulkload_task_failed();
```

Comment:
Does this mean the failed task can never be cleared?

Reply:
Yes. The failed task will cleared by users through fdbcli. I will include this in the next PR.